tests/integration/targets/idrac_reset/tests/_utils.yml

```
@@ -26,6 +26,7 @@
          'ResetAllWithRootDefaults']
         - operation is defined and operation == 'export'
       register: scp_export_before_reset
+      ignore_errors: true
```

why are we ignoring error here?